Tests/TurfTests/GeoJSONTests.swift

```diff
@@ -39,9 +39,9 @@ class GeoJSONTests: XCTestCase {
             ]
         ]
         
-        let polygon = Geometry.Polygon(coordinates: Geometry.PolygonRepresentation(coordinates))
+        let polygon = Geometry.Polygon(coordinates: .init(coordinates))
         let polygonFeature = Feature(polygon)
-        XCTAssertEqual((polygonFeature.geometry.value as! Geometry.PolygonRepresentation).coordinates, coordinates)
+        XCTAssertEqual((polygonFeature.geometry.value as! Polygon).coordinates, coordinates)
```

Comment:
macOS tests are failing to build:

```
CompileSwift normal x86_64 /Users/vagrant/git/Tests/TurfTests/GeoJSONTests.swift (in target 'TurfMacTests' from project 'Turf')
    cd /Users/vagrant/git
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /Users/vagrant/git/Tests/TurfTests/PolygonTests.swift /Users/vagrant/git/Tests/TurfTests/MultiPolygonTests.swift /Users/vagrant/git/Tests/TurfTests/PointTests.swift /Users/vagrant/git/Tests/TurfTests/GeometryCollectionTests.swift /Users/vagrant/git/Tests/TurfTests/BoundingBoxTests.swift /Users/vagrant/git/Tests/TurfTests/MultiLineStringTests.swift /Users/vagrant/git/Tests/TurfTests/FeatureCollectionTests.swift /Users/vagrant/git/Tests/TurfTests/TurfTests.swift /Users/vagrant/git/Tests/TurfTests/LineStringTests.swift /Users/vagrant/git/Tests/TurfTests/Fixture.swift /Users/vagrant/git/Tests/TurfTests/MultiPointTests.swift -primary-file /Users/vagrant/git/Tests/TurfTests/GeoJSONTests.swift -emit-module-path /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/Objects-normal/x86_64/GeoJSONTests\~partial.swiftmodule -emit-module-doc-path /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/Objects-normal/x86_64/GeoJSONTests\~partial.swiftdoc -serialize-diagnostics-path /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/Objects-normal/x86_64/GeoJSONTests.dia -emit-dependencies-path /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/Objects-normal/x86_64/GeoJSONTests.d -emit-reference-dependencies-path /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/Objects-normal/x86_64/GeoJSONTests.swiftdeps -target x86_64-apple-macos10.13 -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -I /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Products/Debug -I /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/lib -F /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Products/Debug -F /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -enable-testing -g -module-cache-path /Users/vagrant/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -swift-version 5 -enforce-exclusivity=checked -Onone -D DEBUG -disable-swift3-objc-inference -serialize-debugging-options -serialize-debugging-options -Xcc -working-directory -Xcc /Users/vagrant/git -enable-anonymous-context-mangled-names -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/TurfTests-generated-files.hmap -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/TurfTests-own-target-headers.hmap -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/TurfTests-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/all-product-headers.yaml -Xcc -iquote -Xcc /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/TurfTests-project-headers.hmap -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Products/Debug/include -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/DerivedSources-normal/x86_64 -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/DerivedSources/x86_64 -Xcc -I/Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/DerivedSources -Xcc -DDEBUG=1 -module-name TurfTests -o /Users/vagrant/Library/Developer/Xcode/DerivedData/Turf-dazvllxwovhcwqbdhgzxlrtuauvv/Build/Intermediates.noindex/Turf.build/Debug/TurfMacTests.build/Objects-normal/x86_64/GeoJSONTests.o
/Users/vagrant/git/Tests/TurfTests/GeoJSONTests.swift:44:59: error: 'Polygon' is ambiguous for type lookup in this context
        XCTAssertEqual((polygonFeature.geometry.value as! Polygon).coordinates, coordinates)
                                                          ^~~~~~~
ApplicationServices.Polygon:1:18: note: found this candidate
public typealias Polygon = MacPolygon
                 ^
Turf.Polygon:1:15: note: found this candidate
public struct Polygon : Equatable {
              ^
```

On macOS, XCTest imports AppKit, which imports ApplicationServices.